docs/user-guide/replicasets.md

```diff
@@ -0,0 +1,91 @@
+---
+---
+
+* TOC
+{:toc}
+
+# Replica Set
+
+## What is a _Replica Set_?
+
+Replica Set is the next-generation Replication Controller. The only difference between a
+_Replica Set_ and a [_Replication Controller_](replication-controller.md) right now is
+the selector support. Replica Set supports the new set-based selector requirements as
+described in the [labels user guide](labels.md#label-selectors) whereas a Replication
+Controller only supports equality-based selector requirements.
+
+Most [`kubectl`](kubectl/kubectl.md) commands that support Replication Controllers also
+support Replica Sets. Commands that support Replica Sets are:
+[`scale`](kubectl/kubectl_scale.md), [`autoscale`](kubectl/kubectl_autoscale.md),
+[`expose`](kubectl/kubectl_expose.md). One exception is the
+[`rolling-update`](kubectl/kubectl_rolling-update.md) command. If you want the rolling
+update functionality please consider using Deployments instead. Also, the
+[`rolling-update`](kubectl/kubectl_rolling-update.md) command is imperative
+whereas Deployments are declarative, so we recommend using Deployments through the
+[`rollout`](kubectl/kubectl_rollout.md) command.
+
+While replica sets can be used independently, today it's mainly used by
+[Deployments](deployments.md) as a mechanism to orchestrate pod creation, deletion and
+updates. When you use Deployments you don't have to worry about manging the replica sets
+that they create. Deployments own and manage their replica sets.
```

Comment:
I moved it up. I want this section to be separate and to have a heading to highlight it. So leaving the heading.